nestkernel/nestmodule.cpp

```diff
@@ -609,6 +609,17 @@ NestModule::CopyModel_l_l_DFunction::execute( SLIInterpreter* i ) const
   const std::string newmodname = getValue< std::string >( i->OStack.pick( 1 ) );
   DictionaryDatum dict = getValue< DictionaryDatum >( i->OStack.pick( 0 ) );
 
+  if ( oldmodname.toString() == "gap_junction" )
```

Comment:
Could you also add a comment pointing out that this is a hotfix, with reference to the relevant issues?